plugins/azure-devops-backend/src/service/router.ts

```diff
@@ -195,6 +187,8 @@ export async function createRouter(
   });
 
   router.get('/readme/:projectName/:repoName', async (req, res) => {
+    const host = config.getString('azureDevOps.host');
+    const organization = config.getString('azureDevOps.organization');
```

Comment:
Good call out, for now this will have to stay. The plan is to follow up with PR(s) to get this passed in instead, just setting the ground work in this PR

Reply:
`azureDevOps.host` and `azureDevOps.organization` are deprecated now right? What happens when a user decides to remove the deprecated settings?

Reply:
Maybe `azureDevOps.host` and `azureDevOps.organization` should not be deprecated just yet? Since you plan to change the route in the next PR(s) these settings can be deprecated as part of those?

Reply:
Alright, I've removed deprecating the config and removed the comment about that from the changeset.